x-pack/solutions/observability/plugins/logs_explorer/public/hooks/use_data_views.tsx

```diff
@@ -53,11 +53,8 @@ const useDataViews = ({ core, dataViewsService, events }: DataViewsContextDeps)
 
   // Test whether a data view can be explored in Logs Explorer based on the settings
   const isDataViewAllowed: IsDataViewAllowed = useCallback(
-    (dataView) =>
-      dataView.testAgainstAllowedList(
-        core.uiSettings.get(OBSERVABILITY_LOGS_EXPLORER_ALLOWED_DATA_VIEWS_ID)
-      ),
-    [core.uiSettings]
+    (dataView) => dataView.testAgainstAllowedList(DEFAULT_ALLOWED_LOGS_BASE_PATTERNS),
```

Comment:
Replaced all setting usages with this constant, the code for Logs Explorer is not used and will be removed anyways but it's outside the scope of this issue.